inst/tests/test-parsers.R

```diff
@@ -156,6 +156,20 @@ test_that("ymd functions correctly parse dates with no separators and no quotes"
               equals(as.POSIXct("2010-01-02 23:59:59", tz = "UTC")))
 })
 
+test_that("numbers are correctly converted into character for parsing", {
+  # numbers with 000000 at the end are wrongly converted by as.character
```

Comment:
They are not wrongly converted. That's the feature which is customized by `scipen` option.